Public/FourKeyMetrics.ps1

```diff
@@ -163,6 +164,24 @@ function global:Get-ReleaseMetrics {
             continue
         }
 
+        if ($previousRelease.IsFix -ne $true){
+            $previousSuccess = $previousRelease
+        }
+
+        if ($thisRelease.IsFix){
+            $nextRelease = $releases[$i+2]
```

Comment:
Not sure what powershell does if you try to address past the end of the array so we probably want a guard clause here otherwise looks good to me

Reply:
I've had a look at this and I'm not worried - the default Powershell behaviour seems to return `$null` when we step out of bounds, `$null` isn't truthy, so everything behaves as I'd expect.